### DISCUSSION TITLE
[FEATURE-REQUEST] Para el default.template.html agregar el campo "logo"

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#223] [3. Ideas & Feature Requests] [answered]
Hola

Actualmente cuando se utiliza el pbemail con el template default.html. Nos hemos percatado que el cliente le gusta ese formato (2 mensajes + titulo), pero siempre nos pide cambiar el logo (actualmente se ve el de cotalker) y nos gustaría que agregaran el campo logo y asi cuando ingresamos una url, podemos reemplazar el logo al que queremos.

Dejo un SS del campo que indico

![image](https://user-images.githubusercontent.com/61419471/120245621-b5917580-c23b-11eb-828c-58c5a415ebf7.png)

Gracias

**Comment by acedward** [comment, marked as answer]
Si no mal recuerdo, la primera versión de este correo (hace como 4 años) automáticamente usaba company.logo o algo así, si es que estaba definido - sino usaba el de cotalker

**Reply by NicolasPLG** [reply]
hola @acedward , entonces si la company tiene un logo se vería. el tema que tengo es que desde el admin lo tengo sin poder ingresarlo (ya que revise company)

![image](https://user-images.githubusercontent.com/61419471/120246192-84b24000-c23d-11eb-85ef-36be17f674ef.png)

Levanté una solicitud interna para agregar ese campo, pero no sé si al agregar el logo de la company funcionaría. Tambien en caso que sea eso, levantaría un bug o feature porque no me permite subir imagenes

**Reply by NicolasPLG** [reply]
@acedward no funcionó sobre el logo de la compañía. voy a pedir apoyo para un faas de esto y asi no tocar el bot.

gracias y cerraré esta discusión

**Reply by acedward** [reply]
@NicolasPLG esto no está bien - ni mal de todas formas.
Revisé la base de código fuente y esta todo listo para poder implementarlo rápidamente.

Si es urgente pide priorizarlo con tu representante Cotalker.

**Reply by NicolasPLG** [reply]
gracias @acedward comentaré a mi representante
